apps/vr-tests/src/stories/Pivot.stories.tsx

```diff
@@ -31,7 +31,7 @@ storiesOf('Pivot', module)
         <PivotItem linkText="My Files" itemCount={42} itemIcon="Emoji2">
           Content
         </PivotItem>
-        <PivotItem itemCount={23} itemIcon="Recent" />
+        <PivotItem itemCount="Six" itemIcon="Recent" />
```

Comment:
Could you change this to something like `"20+"` instead? Seems unlikely that anyone would want to use spelled-out versions of numbers in a real app.

Reply:
True. I was trying to make it super clear that it's a string, but you're right.